spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -98,6 +98,24 @@ public void setTrustedPackages(String[] trustedPackages) {
 		 */
 		private int executorThreads = 4;
 
+		/**
+		 * Properties for {@link com.google.api.gax.retrying.RetrySettings}
```

Comment:
The description of the configuration property must be simple, single like text. No any JavaDoc tags there, please: https://docs.spring.io/spring-boot/docs/2.0.2.RELEASE/reference/htmlsingle/#configuration-metadata-property-attributes